feature-libs/checkout/base/core/facade/checkout-delivery-modes.service.spec.ts

```diff
@@ -212,7 +212,7 @@ describe(`CheckoutDeliveryModesService`, () => {
       );
     });
 
-    // TODO: Replace with event testing once we remove ngrx store.
+    // TODO:#checkout Replace with event testing once we remove ngrx store.
```

Comment:
Shouldn't it be `TODO:#deprecation-checkout` as we're going to remove this after removing ngrx store. Moreover, we're not using #checkout, but instead using #deprecation-checkout

Example: https://github.com/SAP/spartacus/pull/14479/files#diff-be5bc15655bc76f36d02273572519649ee7cafad56caaac87cc7c644010e5185R143

Reply:
this entire file

Reply:
The `deprecation-checkout` is for the future (see todo.md)